### PR TITLE
docs: update minutes of first three weeks of coding period.

### PR DESCRIPTION
**Pull request by k-avy**
## Project

License as a service 

### Changes

Added the report and minutes of first three weeks of coding period.

### CODE REVIEW

File: docs/2023/laas/updates/2023-06-02.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: Coding Period 
+author: Kavya Shukla
+author url: https://github.com/k-avy
```
Comment (avinal) [inline docs/2023/laas/updates/2023-06-02.md:4]:
```suggestion
author_url: https://github.com/k-avy
```
Interesting that build did not fail.

File: docs/2023/laas/updates/2023-06-02.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+tags: [license as a service, open source licensing]
+---
+
```
Comment (avinal) [inline docs/2023/laas/updates/2023-06-02.md:7]: You have to add a licensing text to all the markdown files as well. Please see other's PR.

File: docs/2023/laas/updates/2023-06-09.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+---
+title: Coding Period 
+author: Kavya Shukla
+author url: https://github.com/k-avy
```
Comment (avinal) [inline docs/2023/laas/updates/2023-06-09.md:4]:
```suggestion
author_url: https://github.com/k-avy
```

File: docs/2023/laas/updates/2023-06-09.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+tags: [license as a service, open source licensing]
+---
+
```
Comment (avinal) [inline docs/2023/laas/updates/2023-06-09.md:7]: Add licensing text

File: docs/2023/laas/updates/2023-06-09.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+* Shaheem suggested to discuss it with Anupam and Gaurav for License Candidate file.
+  * For now we will use License database of license.
+* If the data in database is not same as that in json file we will migrate it from
+  json file. For ths we have RF flag in fossology, this flag refers if the data is
+  from UI or from json.
+* Marydone is feild for candidate license flag.
+
+## General
+
+* Do we want plane log or structured log:
```
Comment (avinal) [inline docs/2023/laas/updates/2023-06-09.md:38]:
```suggestion
* Do we want plain log or structured log?
```

File: docs/2023/laas/updates/2023-06-09.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+* database can be populated with json file etc
+
+## Discussions
+
+* Shaheem suggested to discuss it with Anupam and Gaurav for License Candidate file.
+  * For now we will use License database of license.
+* If the data in database is not same as that in json file we will migrate it from
+  json file. For ths we have RF flag in fossology, this flag refers if the data is
+  from UI or from json.
+* Marydone is feild for candidate license flag.
```
Comment (avinal) [inline docs/2023/laas/updates/2023-06-09.md:34]:
```suggestion
* Marydone is field for candidate license flag.
```

**Review by avinal — CHANGES_REQUESTED** [review]
Please use Grammarly/LanguageTool extension to fix grammatical errors. Also format your markdown files.

File: docs/2023/laas/updates/2023-06-16.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+* [Avinal](https://github.com/avinal)
+* [Anupam](https://github.com/ag4ums)
+* [Kavya](https://github.com/k-avy)
+  
+## Discussions
+
+* For update & create shortanme should not be their in parameter of URL.
+* Add error for invalid URL.
+* SPDX id will aso be a required feild in create.
```
Comment (GMishx) [inline docs/2023/laas/updates/2023-06-16.md:29]:
```suggestion
* SPDX id will also be a required field in create.
```

File: docs/2023/laas/updates/2023-06-16.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+* Get all licenses:
+  
+        to get all the details of the licenses present in the database.
+        gets the data of all the data components present in the database.
+* Get licenses by shortname:
+  
+        to get the data of the license with the shortname passed as a parameter.
+        error when no such license found.
+
+## COnclusion and future plan
```
Comment (GMishx) [inline docs/2023/laas/updates/2023-06-16.md:46]:
```suggestion
## Conclusion and future plan
```

**Review by GMishx — CHANGES_REQUESTED** [review]

**Review by GMishx — APPROVED** [review]

**Pull request merged**